libbrowser/src/libbrowser_cef.cpp

```diff
@@ -1323,16 +1330,28 @@ void MCCefBrowserBase::SetOverflowHidden(bool p_hidden)
 	MCCStringFree(t_overflow_script);
 }
 
-bool MCCefBrowserBase::GetScrollbars(void)
+bool MCCefBrowserBase::GetVerticalScrollbarEnabled(void)
+{
+	// IM-2014-08-25: [[ Bug 13272 ]] Show / hide scrollbars by setting the overflow style to empty / "hidden".
+	return !GetOverflowHidden(kMCCefScrollbarVertical);
+}
+
+void MCCefBrowserBase::SetVerticalScrollbarEnabled(bool p_scrollbars)
 {
 	// IM-2014-08-25: [[ Bug 13272 ]] Show / hide scrollbars by setting the overflow style to empty / "hidden".
-	return !GetOverflowHidden();
+	/* UNCHECKED */ SetOverflowHidden(kMCCefScrollbarVertical, !p_scrollbars);
```

Comment:
I mean in executing the javascript. Is there any point in checking for a failure that won't happen?
Also, don't forget that this is supposed to be an independent library which doesn't link to libfoundation